backend/.golangci.yml

```diff
@@ -26,6 +26,9 @@ linters:
 
 # all available settings of specific linters
 linters-settings:
+    errcheck:
+        check-blank: true
+
```

Comment:
`check-type-assertions` is a useful one - i've bumped into that in the `otel.go` file (casting interfaces and not checking error)